### PR TITLE
 Folder restructuring 

### PR DESCRIPTION
**Pull request by jvnaiduwin**

### CODE REVIEW

**Review by giorgiozoppi — COMMENTED** [review]
Can we keep vcpkg support?

**Comment by MelamudMichael** [comment]
> Can we keep vcpkg support?

Hi @giorgiozoppi 

For now we are going to support only redhat and yocto - so no need to vcpkg

**Review by MelamudMichael — APPROVED** [review]

**Comment by jvnaiduwin** [comment]
> Can we keep vcpkg support?

As we have discussion with Michael, and he discussed with Steven we agreed upon 

"[Tuesday 1:29 PM] Michael Melamud

J Vignesh Naidu (C)  no need for vcpkg, we are supporting redhat and yocto at the moment 

[Tuesday 1:29 PM] Michael Melamud

this was the agreement with steven"

**Review by linsonittoop — APPROVED** [review]

**Review by stevenhartley — CHANGES_REQUESTED** [review]
I pulled the project and built per the README and it fails to find RapidJSON even thought I installed it with vcpkg.  Can you please check what is wrong?  The build is complaining of a missing RapidJSONConfig.cmake or rapidjson-config.cmake file

File: uprotocol/tools/tools.h
Comment (stevenhartley) [inline uprotocol/tools/tools.h:1]: Would this not be provided by some other external library?

File: uprotocol/proto/CMakeLists.txt
Comment (stevenhartley) [inline uprotocol/proto/CMakeLists.txt:1]: We should not include the cloudevent proto here, it should be pulled as an external dependency for the cloudevent builders.

**Review by stevenhartley — CHANGES_REQUESTED** [review]
additional comments

**Review by stevenhartley — APPROVED** [review]
Change is now good and we can merge to the 1.5 branch

**Pull request merged**